graf2d/asimage/src/TASImage.cxx

```diff
@@ -1095,7 +1095,8 @@ void TASImage::FromPad(TVirtualPad *pad, Int_t x, Int_t y, UInt_t w, UInt_t h)
 
    if (gROOT->IsBatch()) { // in batch mode
       TVirtualPS *psave = gVirtualPS;
-      gVirtualPS = (TVirtualPS*)gROOT->ProcessLineFast("new TImageDump()");
+      const static auto tclass_TImageDump = TClass::GetClass("TImageDump");
```

Comment:
Why not have the `asimage` depends on the always-on `postscript` package and simply use `#include "TImageDump.h"` ?
```suggestion
      const static auto tclass_TImageDump = TClass::GetClass("TImageDump");
```
Technically I guess that if the simple solution is not possible, we ought to also add something along this line:
```suggestion
      const static auto initializer = []() { 
          auto ret = TClass::GetClass("TImageDump");
          if (!ret)
              Fatal("FromPad", "Could not find the TClass for TImageDump");
      };
      const static auto tclass_TImageDump = initializer();
```

Reply:
So, this is clearly the simplest solutions of all: include a header and link. I tested that and it works.
However, are we sure it does not have side effects? Why was this there in the first place?
Rolling back is not too difficult given the discussions above, almost everything is better than interpreting a string of code at runtime in this context.

Reply:
Yes, the `postcript` module is always present and we are not here leveraging any run-time configuration, so the indirection does not bring any benefit.